exp/operator/api/v1alpha4/groupversion_info.go

```diff
@@ -25,10 +25,10 @@ import (
 )
 
 var (
-	// GroupVersion is group version used to register these objects
+	// GroupVersion is group version used to register these objects.
```

Comment:
Question: if I look at the examples in the proposal, we are using v1alpha1 for this api group. is there  a reason to jump to v1alpha4?

Reply:
Hm, I missed that somehow. Should we stick with v1alpha4 so we can avoid confusion in the future?

Reply:
My major concern about linking this new API to the other APIs in CAPI is the fact that we will get to v1beta1 pretty quickly without having time to gather enough user feedback.
@vincepri @detiber @CecileRobertMichon opinions?

Reply:
We can start from v1alpha1 for the operator